controllers/articles.controllers.js

```diff
@@ -1,12 +1,23 @@
-const { selectArticle } = require("../models/articles.models");
+const { selectArticleById, selectArticles } = require("../models/articles.models");
 
 exports.getArticleById = (req, res, next) => {
-    const { article_id } = req.params
-    selectArticle(article_id)
-    .then((article) => {
-        res.status(200).send(article);
-      })
-      .catch((err) => {
-        next(err)
-      })
-}
\ No newline at end of file
+    const { article_id } = req.params;
+    selectArticleById(article_id)
+        .then((article) => {
+            res.status(200).send(article);
+        })
+        .catch((err) => {
+            next(err);
+        });
+};
+
+exports.getArticles = (req, res, next) => {
+    selectArticles()
+        .then((articles) => {
+
+            res.status(200).send(articles);
```

Comment:
please send articles as a property on the body rather than the body itself.   ie {articles}